### DISCUSSION TITLE
No authenticator for 'xxx'

### DISCUSSION THREAD

**Discussion by Nsbx** [ValvePython/steamctl#33] [Q&A] [answered]
Hi :)

I have some trouble to use command to retrieve my qrcode.
When i execute the command i get "No authenticator for 'xxx'" BUT
If i execute "authenticator status" it work perfectly and ask for my password.

What is the problem ?

Thank

**Comment by rossengeorgiev** [comment, marked as answer]
You have to first add an authenticator. If you already have one, like the Steam Mobile App, then you need to remove it, and add via `steamctl`. During the process you can setup both Steam Mobile App and `steamctl` with the same authenticator secret. Just follow the instructions.

**Reply by Nsbx** [reply]
Oh i see, i tried but actually steam have some trouble to send mail ...
I keep you updated when it solved and if I have succeeded
Thank

**Reply by Nsbx** [reply]
Oh god i'm fucking dumb ...
I configure some mail rule and i doesn't see the mail ><"
After reminder that, this is perfect 👍 

Thank for your help
